.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.16
+      - image: cimg/go:1.18
 
     working_directory: /go/src/github.com/warrensbox/terraform-switcher
```

Comment:
The working directory is not right here:
In the previous version, it was `/go/` for image: circleci/golang:1.16

It should be `/home/circleci/go` for the new image: cimg/go:1.18